docs/README.md

```diff
@@ -1,21 +1,21 @@
-# Overview
+# Introduction
 
-react-map-gl is a react friendly API wrapper around
+react-map-gl is a suite of React components for
 [Mapbox GL JS](https://github.com/mapbox/mapbox-gl-js).
 
 ## Example
 
 ```js
-import MapGL from 'react-map-gl';
+import ReactMapGL from 'react-map-gl';
```

Comment:
I like to not rename things from the package wherever possible, to prevent confusions later if you needed to `grep` for anything.